src/Livewire/Order/Order.php

```diff
@@ -46,7 +47,8 @@ class Order extends Component
 {
     use Actions, CreatesDocuments, WithTabs;
 
-    protected string $view = 'flux::livewire.order.order';
+    #[Locked]
+    public string $view = 'flux::livewire.order.order';
```

Comment:
why change visibility to public?

Reply:
Because protected is resetted to default After Rendering. Mount gets called only once.